packages/components/table/examples/TableWithSortingPropsExample.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from 'react';
+import { Badge, Table, TableCellSorting } from '@contentful/f36-components';
+
+export default function TableWithSorting() {
+  const contentTypes = [
+    {
+      id: '1',
+      name: 'Category',
+      description:
+        'Categories can be applied to Courses and Lessons. Assigning Multiple categories is also possible.',
+      updatedAt: 'Nov 15, 2021',
+      status: 'published',
+    },
+    {
+      id: '2',
+      updatedAt: 'Nov 15, 2021',
+      status: 'draft',
+    },
+    {
+      id: '3',
+      name: 'Layout',
+      description:
+        'A page consisting of freely configurable and rearrangeable content modules.',
+      updatedAt: 'Nov 15, 2021',
+      status: 'published',
+    },
+  ];
+
+  const [sorting, setSorting] = useState(undefined);
+  const [sortedRows, setSortedRows] = useState(contentTypes);
+
+  const handleSort = (direction) => {
+    setSortedRows((rows) => {
```

Comment:
Sorting logic doesn't work for me on the Table sorting example preview, it doesn't sort the column items in the right order.

Reply:
I missed a place for the enum rename. Fixed.